### DISCUSSION TITLE
ag库里报错CheckGLError ：1282

### DISCUSSION THREAD

**Discussion by kang12** [Tencent/libpag#2377] [Q&A] [answered]
请问下 看日志打印  pag库里报错CheckGLError ：1282  应该是渲染环境异常   这个应用层能加回调监听这个报错吗 应用层需要监听处理这个异常给到用户提示

**Comment by kevingpqi123** [comment, marked as answer]
这里添加监听的必要性不大，即便监听到业务层好像也做不了什么，如果出现这种报错，是需要根据业务场景修改该类问题， 这类问题是 bug 是需要修复解决，而不是规避
pag 侧能够做到的是打印关键错误日志，且保证不 crash
